.vscode/settings.json

```diff
@@ -6,4 +6,4 @@
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
```

Comment:
why is this change needed?

Reply:
It is not needed. My editor inserts final EOL when saving files. I have removed it.